consumers/hidden-test-consumer/src/hidden_test_consumer/protos/hidden_test_process.proto

```
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package hidden_test_process;
+
+service HiddenTestProcess {
+  rpc streamHiddenTestProcess (stream ProcessRequest) returns (ProcessResponse) {}
```

Buf (1.47.2)

RPC name "streamHiddenTestProcess" should be PascalCase, such as "StreamHiddenTestProcess".

(RPC_PASCAL_CASE)